app/src/main/java/com/kickstarter/ui/viewholders/ProjectViewHolder.kt

```diff
@@ -513,9 +514,10 @@ class ProjectViewHolder(
             .compose(bindToLifecycle())
             .compose(observeForUI())
             .subscribe { url: String? ->
-                url?.let {
+                binding.projectSocialImage.isGone = url.isNullOrEmpty()
+                if (!url.isNullOrEmpty()) {
```

Comment:
could the ViewModel just emit a non optional string url, instead of checking that logic on the ViewHolder

Reply:
we do the issue if its empty i i will remove the optional check